### DISCUSSION TITLE
Add `zip` extension

### DISCUSSION THREAD

**Discussion by JDMF19** [NativePHP/laravel#353] [Ideas]
I am trying to use *maatwebsite/excel* package to import Excel files
but it has a dependency on php_zip extension and when I build de app don't work

Is Zip Extension supported or I'm doing something wrong? 

Please help me.

**Comment by simonhamp** [comment]
The current builds of PHP that we ship don't have the `zip` extension, but it is supported, we just need to build it in.

This is definitely coming in a future build, so look out for it soon.

Moving this to ideas as it's really a feature request.

**Comment by simonhamp** [comment]
This has been added as of [php-bin 0.5.0](https://github.com/NativePHP/php-bin/releases/tag/0.5.0)
